app/src/main/java/org/mozilla/focus/session/ui/SessionsSheetFragment.java

```diff
@@ -98,14 +99,17 @@ public void onAnimationEnd(Animator animation) {
 
     /* package */ Animator animateAndDismiss() {
         final Animator animator = playAnimation(true);
+        final MainActivity activity = (MainActivity)getActivity();
```

Comment:
\[Checkstyle\] ERROR: 'typecast' is not followed by whitespace\.

Reply:
I wonder why you are doing this here and not in onAnimationEnd()? With this we are keeping a reference to the activity the whole time unnecessarily.

Also the cast to MainActivity does not seem like something we need to do.

Reply:
@pocmo Shifted the activity reference to inside onAnimationEnd().
Also, without the typecasting, getSupportFragmentManager() is unable to get resolved, because it requires a reference to an AppCompatAcitivty. I can either typecast to an AppCompatAcitivty or MainActivity. I've chosen the latter since it extends AppCompatAcitivty.